webnn-status.md

```diff
@@ -445,11 +751,19 @@ The total number of WebNN ops is 78. This table currently lists ops that are imp
             <th id="wpt_total"></th>
             <th id="x_total" colspan="2"></th>
             <th id="dml_total" colspan="2"></th>
+            <th id="mlservice_total" colspan="2"></th>
+          </tr>
+        `;
+
+         oplistFramework = oplistFramework + `
+          <tr>
+            <th id="spec_total"></th>
```

Comment:
This should display "78 / 78, 100.0%". Currently "156 / 78, 200%".

The spec_total formula needs tweaking. 200% looks cool though  :smile: :rocket: